_posts/2023-09-29-installing-hms-core-in-as-emulator.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Installing HMS Core in Android Studio Emulator"
+excerpt: "If you want to test your HMS implementation and you do not have a Huawei device, it is easy to mock it and test these services in the Android Studio emulator."
+tags: Mobile Android Huawei HMS
+authors:
+- Edgar Barragan
+header:
+  teaser: /assets/images/post/hms-core/hms_core_banner.png
+  teaser_alt: Huawei logo
+category: Android
+---
+
+![](/assets/images/post/hms-core/hms_core_banner.png)
+
+# Instroduction
+
+This is a quick guide with few and simple instructions on installing HMS Core in Android Studio Emulator. If you want to test your HMS implementation and you do not have a Huawei device, this is your best option.
+
+# Steps
+
+**1.** Create a new device with only Google APIs, no Play Store. For this example, a Pixel 4a with API 31 is being used.
+
+Choose a device without Play Store.
+
+![](/assets/images/post/hms-core/hms_core_step_1.png)
+
+Choose a version of Android with Google APIs.
+
+![](/assets/images/post/hms-core/hms_core_step_1.1.png)
+
+**2.** Install the Huawei AppGallery APK
+
+You can either Google or install it from the [Huawei Consumer website](http://consumer.huawei.com/en/mobileservices/appgallery/).
+
+Download the AppGallery APK.
+
+<div style="margin:auto;text-align:center">
+<img src="/assets/images/post/hms-core/hms_core_step_2.png" height="600"/>
+</div><br>
```

Comment:
This fixed height is likely to cause issues, I'd recommend to use the figure component:
```suggestion
{% include components/figure.html url="/assets/images/post/hms-core/hms_core_step_2.png" %}
```